CWE/events/italy.txt

```diff
@@ -455,6 +455,1043 @@ country_event = {
+		any_pop = { limit = { 
+		    OR = { region = PAP_752 region = SIC_753 region = SIC_754 region = SIC_763 region = SAR_765 } 
+			}
+			ideology = { factor = 0.04 value = traditionalist } 
+		}
+		
+		add_country_modifier = { name = political_strife duration = 365 }
+		
+		ai_chance = { factor = 0.1 }
+	}
+   
+}
+
+#Enrico Mattei and ENI
+country_event = {
+  	id = 8007751
+  	title = EVT_8007751_NAME
+  	desc = EVT_8007751_DESC
+  	picture = "enrico_mattei"
+	
+  	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1953  
+		NOT = { year = 1990 }
+		
+		OR = { government = democracy government = hms_government }
+		is_vassal = no
+		war = no
+		
+		#Socialists & Progressives are not in power
+		NOT = { OR = { ruling_party_ideology = communist ruling_party_ideology = communist_social ruling_party_ideology = socialist ruling_party_ideology = progressive } }
+		
+		#Cold War ongoing
+		has_global_flag = blocsenabled
+	}
+
+	mean_time_to_happen = { months = 2  }
+
+	option = {
+		name = EVT_8007751_A
+		
+		petroleum = 1000
+		money = 500000
+		
+		random_country = {
+                       limit = { tag = USA government = democracy is_vassal = no }
+                       relation = { who = THIS value = -50 }
+        	}
+		random_country = {
+                       limit = { tag = ENG government = hms_government is_vassal = no }
+                       relation = { who = THIS value = -50 }
+        	}
+		random_country = {
+                       limit = { tag = NET government = hms_government is_vassal = no }
+                       relation = { who = THIS value = -25 }
+        	}
+		random_country = {
+                       limit = { tag = PER OR = { government = hms_government government = absolute_monarchy } is_vassal = no }
+                       relation = { who = THIS value = 50 }
+        	}
+		random_country = {
+                       limit = { tag = EGY government = presidential_dictatorship is_vassal = no }
+                       relation = { who = THIS value = 50 }
+        	}
+		random_country = {
+                       limit = { tag = JOR OR = { government = hms_government government = absolute_monarchy } }
+                       relation = { who = THIS value = 50 }
+        	}
+		random_country = {
+                       limit = { capital = 1731 OR = { government = hms_government government = absolute_monarchy } } #Lybia
+                       relation = { who = THIS value = 50 }
+        	}
+        random_country = {
+                       limit = { tag = MOR OR = { government = hms_government government = absolute_monarchy } }
+                       relation = { who = THIS value = 50 }
+        	}
+        random_country = {
+                       limit = { tag = TUN OR = { government = hms_government government = absolute_monarchy } }
+                       relation = { who = THIS value = 50 }
+        	}
+        random_country = {
+                       limit = { tag = LEB government = democracy  }
```

Comment:
Please insert a check for is_vassal = no.

Reply:
See above